x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/index/mapper/PointFieldMapperTests.java

```diff
@@ -428,8 +428,10 @@ protected IngestScriptSupport ingestScriptSupport() {
     }
 
     @Override
-    protected boolean supportsColumnAtATimeReader(MapperService mapper, MappedFieldType ft) {
-        // Currently ESQL support for cartesian_point is limited to source values
-        return false;
+    protected BlockReaderSupport getSupportedReaders(MapperService mapper, String loaderFieldName) {
+        // TODO: Support testing both reading from source as well as reading from doc-values
```

Comment:
The problem here is that the entire test class `PointFieldMapperTests` has not been setup for synthetic source tests, which `GeoPointFieldMapperTests` has, and this support is needed for the relevant tests. We can investigate adding this.

Reply:
Done - https://github.com/elastic/elasticsearch/issues/104504